### DISCUSSION TITLE
Any way to output latex table in a two-column document?

### DISCUSSION THREAD

**Discussion by yl4070** [rstudio/gt#1548] [Q&A]
Currently, the latex output depends on the `longtable` package, if it's used in a two column format, although it can still render the table, will result in an error: `longtable` is not used in one-column format. 

Is there any option to not use `longtable` package or any workaround to avoid the latex error while keeping the two column format (not temporally switch to one column format)?

**Comment by hlageek** [comment]
It is extremely hacky, but I worked around `longtable` like so:
```
cat(gsub("longtable", "tabular", gt::as_latex(gt_table)))
```
It really would be nice if `as_latex()` were more customizable.

**Reply by olivroy** [reply]
In the dev version of gt, we output as tabular by default

**Comment by nielsbock** [comment]
I've had the issue with tables not being able to span over two columns as well. @olivroy I think I have a fairly simple solution for this for floating tables by setting `begin{table}` and `end{table}` to `begin{table*}` and `end{table*}`. Adding an argument to `tab_options()` such as `latex.span_multicols = TRUE` which would add the asterisk and only apply if `latex.use_longtable = FALSE`. Would this be of interest?
